src/components/PageElements/Elements/ReviewInlineNewResponse.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+import { Button, Form, Radio, TextArea } from 'semantic-ui-react'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import { SummaryViewWrapperProps } from '../../../formElementPlugins/types'
+import { ReviewResponseDecision } from '../../../utils/generated/graphql'
+import { useCreateReviewResponseMutation } from '../../../utils/generated/graphql'
+import { ReviewDetails } from '../../../utils/types'
+
+interface ReviewInlineNewResponseProps {
+  setIsActiveEdit: (_: boolean) => void
+  summaryViewProps: SummaryViewWrapperProps
+  reviewInfo: ReviewDetails
+  stageNumber: number
+}
+
+/*
+This component is visually the same as "ReviewInlineInput", but is used when reviewing elements that don't have an applicantResponse. Instead of updating a review response, it needs to create a new review response along with its associated applicationResponse.
+*/
+const ReviewInlineNewResponse: React.FC<ReviewInlineNewResponseProps> = ({
+  setIsActiveEdit,
+  summaryViewProps,
+  reviewInfo,
+  stageNumber,
+}) => {
+  const { strings } = useLanguageProvider()
+  const [reviewResponse, setReviewResponse] = useState<{
+    comment?: string
+    decision?: ReviewResponseDecision
+  }>({})
+
+  const [createResponse] = useCreateReviewResponseMutation({
+    onError: (error) => {
+      // setError(error)
+    },
+  })
+
+  // Values required for creating new applicationResponse and reviewResponse
+  const templateElementId = summaryViewProps.element.id as number
+  const applicationId = summaryViewProps.applicationData?.id as number
+  const reviewId = reviewInfo.id
+  const timeSubmitted = summaryViewProps.applicationData?.current.timeStatusCreated
+
+  const reviewOptions = [
+    {
+      label: strings.LABEL_REVIEW_APPROVE,
+      decision: ReviewResponseDecision.Approve,
+    },
+    {
+      label: strings.LABEL_REVIEW_RESSUBMIT,
+      decision: ReviewResponseDecision.Decline,
+    },
+  ]
+
+  const submit = async () => {
+    await createResponse({
+      variables: {
+        templateElementId,
+        applicationId,
+        reviewId,
+        stageNumber,
+        decision: reviewResponse.decision as ReviewResponseDecision,
+        comment: reviewResponse.comment,
+        timeSubmitted,
+      },
+      // The order of these refetch queries appears to be important!
+      refetchQueries: ['getAllResponses', 'getReviewResponses'],
```

Comment:
@nmadruga This is what I needed to fix the "Decision" not updating, and the order seems to be important!!

Reply:
Oh, cool! Yes, I think is because that's the same order `ReviewPageWrapper` does it